README.md

```diff
@@ -17,41 +17,91 @@ Please visit the [latest stable release](https://github.com/Open-CMSIS-Pack/cpac
 `cpackget` installs packs via a CLI interface and it's intended to be easy to understand it. Here is how the command line looks like:
 
 ```bash
-$ cpackget pack|pdsc add|rm <pack-path>
-
-Options:
-
-  -h, --help        Help for cpackget.
-  -V, --version     Outout the version number of cpackget and exit.
-  -v, --verbose     Set verbosiness: None (Errors only), -v (Info) and -vv (Debugging)
+Usage:
+  cpackget [command] [flags]
+
+Available Commands:
+  help        Help about any command
+  index       Updates public index
+  init        Initializes a pack root folder
+  pack        Adds/Removes Open-CMSIS-Pack packages
+  pdsc        Adds or removes Open-CMSIS-Pack packages in the local file system via PDSC files.
+
+Flags:
+  -h, --help               help for cpackget
+  -R, --pack-root string   Specifies pack root folder. Defaults to CMSIS_PACK_ROOT environment variable
+  -q, --quiet              Run cpackget silently, printing only error messages
+  -v, --verbose            Sets verboseness level: None (Errors + Info + Warnings), -v (all + Debugging). Specify "-q" for no messages
+  -V, --version            Prints the version number of cpackget and exit
 
 Use "cpackget [command] --help" for more information about a command.
 ```
 
 For example, if one wanted help removing a pack, running `cpackget pack rm --help` would print out useful information on the subject.
 
-### Examples
+
+### Sepecifying the working pack root folder
+
+If cpackget is going to work on an existing pack root folder, there are two ways to specify it:
+
+1. `export CMSIS_PACK_ROOT=path/to/pack-root; cpackget pack add ARM.CMSIS`
+2. `cpackget --pack-root path/to/pack-root pack add ARM.CMSIS`
+
+The first example is more common because exporting the environment variables can happen only once.
+
+But cpackget is also capable of creating a new pack root folder if needed. For example:
+
+```
+$ cpackget init --pack-root path/to/new/pack-root https://keil.com/pack/index.pidx
+```
+
+The command will create a folder called `path/to/new/pack-root`, a few subfolders (`.Download`, `.Local`, `.Web`)
```

Comment:
a few subfolders -> the following subfolder and then list without brackets.

Reply:
ack